### PR TITLE
Features/account/support 24 phrases

### PR DESCRIPTION
**Pull request by thuyenBitmark**

### CODE REVIEW

File: BitmarkRegistry/Controllers/Main/Properties/PropertiesViewController.swift [hunk 1]
```diff
@@ -233,7 +231,10 @@ extension PropertiesViewController {
     let welcomeLabel = CommonUI.pageTitleLabel(text: "WELCOME TO BITMARK!")
     welcomeLabel.textColor = .mainBlueColor
 
-    let descriptionLabel = CommonUI.descriptionLabel(text: "Register, track, and trade property rights for your digital assets.")
+    let descriptionLabel = CommonUI.descriptionLabel(text:
```
Comment (anhnguyenbitmark) [inline BitmarkRegistry/Controllers/Main/Properties/PropertiesViewController.swift]: We can use `Multiline String Literals` for this. See: https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html

File: BitmarkRegistry/Controllers/Onboarding/LoginViewController.swift [hunk 2]
```diff
@@ -41,21 +51,40 @@ class LoginViewController: BaseRecoveryPhraseViewController {
...
+    _numericOrders = nil
+    clearForm()
+    recoveryPhraseCollectionView.reloadData()
+  }
+
   @objc func tapToSubmit(_ sender: UIButton) {
     do {
-      let account = try AccountService.getAccount(phrases: getUserInputPhrases())
+      let userInputPhrases = getUserInputPhrases()
+      guard let seedVersion = SeedVersion(numberOfPhrases: userInputPhrases.count) else {
```
Comment (anhnguyenbitmark) [inline BitmarkRegistry/Controllers/Onboarding/LoginViewController.swift]: I suggest to put throw in init function.

Comment (thuyenBitmark) [reply BitmarkRegistry/Controllers/Onboarding/LoginViewController.swift]: that's ok. but as I research in docs.swift that init? is Failable Initializer
```To cope with initialization conditions that can fail, define one or more failable initializers as part of a class, structure, or enumeration definition. You write a failable initializer by placing a question mark after the init keyword (init?).```
Reference: https://docs.swift.org/swift-book/LanguageGuide/Initialization.html#//apple_ref/doc/uid/TP40014097-CH18-XID_339 - Failable Initializers
Thanks

Comment (anhnguyenbitmark) [reply BitmarkRegistry/Controllers/Onboarding/LoginViewController.swift]: I don't find any strong clue to state that we should not use throwing initialization. Most likely it's to be compatible with Objective-C version, which does not have throw feature. I'm ok with this approach.

**Pull request merged**